stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,106 @@
+- the _secret word_ (even if it was guessed correctly);
+- a _"play again"_ button.
+
+9. When the user clicks on "play again", the game _starts over_:
+
+- the gallows is _empty_;
+- the question is _changed_;
+- the underscores are _updated_ according to the new secret word.
+- an incorrect guesses counter is _set to 0_.
+
+### Approximate appearance
+
+_This example is purely illustrative. Feel free to change the layout as you wish._
+![screenshot](hangman.png)
+
+### Helpful materials
+
+At the [following link](https://www.figma.com/file/ug2NAUiXPpaFDvch5TWUxd/Hangman-game?type=design&node-id=0%3A1&mode=design&t=4Gj7Ayo0yckppNo8-1) you will find:
+
+- examples of the game flow,
+- body parts (for those who don't want to design them by themselves).
+
+### Main functional requirements
+
+- The design must be _adaptive (or responsive)_, that includes desktop(1440px <= width), tablet(768px <= width < 1440px) and mobile(360px <= width < 768px). When switching between versions everything must be displayed correctly, all functionality must be present, no content must disappear or be left beyond the screen.
+- Initially, body in the index.html file _must be empty_, all necessary elements are generated using JS.
+- At the beginning of the game, the question-answer pair must be _chosen randomly_.
+- The application can be done _in English or in Russian_.
+
+## Repository requirements
+
+- The task must be done in private school's repository.
+- The code must be committed to a separate branch named as the task.
+- The `main` (or old style master) branch must be empty (contain only files like README.md or .gitignore).
+- Commit messages must follow the [guideline](https://docs.rs.school/#/git-convention)
+- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
+- The pull request name must contain **the task name**.
+- The pull request description must be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
+
+## Technical requirements
+
+- The application is displayed correctly and functions properly in the latest version of Google Chrome.
+- It is **not allowed** to use:
+  - CSS frameworks (e.g., `Bootstrap`),
+  - JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.),
+  - Outdated libraries (e.g. `JQuery`, etc.),
+  - `TypeScript`.
+- You can use CSS preprocessors (`SASS`, `SCSS`) and `normalize.css`.
+- Using JS frameworks (e.g., Angular, React, Vue, etc.) is not allowed.
+
+## CrossCheck Criteria (150 points)
+
+_It is recommended to print the secret word in the browser's console to facilitate the cross-check process._
+
+- [ ] Responsive/adaptive UI from 1440px to 360px viewport: `+10`
+- [ ] The generation of DOM elements is implemented. `body` in the `index.html` is empty (can contain only `script` tag): `+20`
```

Comment:
Maybe we should provide few words about how to check if body is empty or not? I guess it's the first task on current stage with such rule and it'll be a lot of questions and mistakes. It's proposal for discussion

Reply:
done